src/binary-search-tree.js

```diff
@@ -5,6 +5,42 @@ class BinarySearchTree {
     this.left = null;
     this.right = null;
   }
+
+  insert(value) {
+    const search = (n) => {
+      if (n.value >= value) {
+        if (n.left === null) {
+          return n.left = new BinarySearchTree(value);
+        }
+        return search(n.left);
+      }
+      if (n.right === null) {
```

Comment:
Start getting in the habit of using the `!` operator when you want to check if something is undefined or null. It is more concise and more flexible than checking if something is strictly just undefined.